WalletWasabi/Wallets/Wallet.cs

```diff
@@ -250,6 +250,7 @@ public override async Task StartAsync(CancellationToken cancel)
 					{
 						// Continue wallet synchronization in the background for all keys skipped by TurboSync.
 						FinalSynchronizationTask = Task.Run(() => PerformFinalSynchronizationAsync(cancel), cancel);
+						_ = FinalSynchronizationTask.ContinueWith(HandleExceptionForFinalSynchronization, TaskContinuationOptions.OnlyOnFaulted);
```

Comment:
Can't we merge these lines together?

```suggestion
						FinalSynchronizationTask = Task.Run(() => PerformFinalSynchronizationAsync(cancel), cancel)
						.ContinueWith(HandleExceptionForFinalSynchronization, TaskContinuationOptions.OnlyOnFaulted);
```